shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1190,12 +1185,19 @@ - (void)updateViewportPadding {
 
 #pragma mark - Keyboard events
 
+// using keyboardWillChangeFrame instead of keyboardWill(Show/Hide) because of split/floating
```

Comment:
The comment says "keybaordWill(show/hide), but we don't have a "keyboardWillShow", maybe just "keyboardWillBeHidden"?

Reply:
keyboardWillShow/keyboardWillHide are part of the UIKit framework; so keyboardWillBeHidden is not appropriated here and my comment is still relevant 🙂
BTW I tried to answer to you in my opened issue last week but I could not without closing the issue.
(nothing related but I loved [the document](https://docs.google.com/document/d/1V7Jc_RGaknrBBPPBBKB8lT7f3PKhYr8sin35MSMFAf4/) you wrote related to blurs... so interesting to read !)